### ISSUE TITLE
CutMix: Regularization Strategy to Train Strong Classifiers with Localizable Features

### ISSUE THREAD

**Issue by llien30** [mn1204/paper_summary#52]
# INFO

## Author
Sangdoo Yun, Dongyoon Han, Seong Joon Oh,Sanghyuk Chun, Junsuk Choe, Youngjoon Yoo
## Conference or Year
ICCV2019 oral
## Link
- [arXiv](https://arxiv.org/abs/1905.04899)
- [official GitHub](https://github.com/clovaai/CutMix-PyTorch)

# Abstract
### proposed the **CutMix** augumentation strategy
* patches are cut and pasted among training images
* the ground truth labels are also mixed proportionally to the area of the patches

<img width="512" alt="Screen Shot 2020-05-21 at 15 53 25" src="https://user-images.githubusercontent.com/51707667/82531910-36bf2280-9b7b-11ea-8420-a13c57fb386c.png">

# Proposed Method
<img width="364" alt="Screenshot 2020-05-21 17 04 09" src="https://user-images.githubusercontent.com/51707667/82537520-214ef600-9b85-11ea-8f4b-ab23d5cf17d4.png">
<img width="529" alt="Screen Shot 2020-05-21 at 17 05 13" src="https://user-images.githubusercontent.com/51707667/82537599-3deb2e00-9b85-11ea-8f9b-c05bb9ae427a.png">

# Evaluation

# Contribution

# Discussion, Future Work

## Date
2020/05/21
